infra/core/host/appservice.bicep

```diff
@@ -33,6 +33,13 @@ param numberOfWorkers int = -1
 param scmDoBuildDuringDeployment bool = false
 param use32BitWorkerProcess bool = false
 
+// Target DB properties
+param connectionStringKey string = 'AZURE-SQL-CONNECTION-STRING'
```

Comment:
We wouldn't update /core/host/appservice to use SQL because the app service itself can be used with any DB and it shouldn't default to anything.

Reply:
update default to empty